packages/create-waku-app/.gitignore

```diff
@@ -0,0 +1,2 @@
+examples
+package-lock.json
```

Comment:
OK, if there are problems with those examples I can propose to make an issue to my name to revamp them. 
In general I think it is a good idea to move back to npm at least because it's quirks are supported by many other package managers. 

#164